client/state/selectors/get-user-devices.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get, values } from 'lodash';
+
+/**
+ * Returns the current user's devices
+ *
+ * @param  {Object} state Global state tree
+ * @return {Object}       current user's devices
+ */
+export default function getUserDevices( state ) {
+	return values( get( state, 'userDevices.items', null ) );
```

Comment:
we probably don't need `null` here but it's no big deal. I tested with `values()` and it returns `[]` given `null` or `undefined` either way.